dev/com.ibm.ws.security.oidc.server_fat.claimPropagation/fat/src/com/ibm/ws/security/openidconnect/server/fat/claimPropagation/CommonTests/BasicIdTokenClaimPropagationTests.java

```diff
@@ -0,0 +1,766 @@
+        setParmValues(updatedTestSettings.getUserName());
+
+        List<validationData> expectations = vData.addSuccessStatusCodes(null);
+        expectations = setGroupIdsExpectations(expectations);
+        expectations = setTestParmExpectations(expectations, allUniqueIntermedParms, thirdPartyPropagatedIdTokenFalse);
+        expectations = setRealmNameExpectations(expectations);
+        expectations = setIssExpectations(expectations, appName);
+        expectations = setAudExpectations(expectations, appName);
+        genericRP(_testName, wc, updatedTestSettings, steps, expectations);
+    }
+
+    /**
+     * The Third party OP will populate the idToken that it creates with the groupIds claim (by default)
+     * The Third party OP does not populate the idToken that it creates with the any extra claims
+     * The intermediate OP does have thirdPartyIDTokenClaims configured with "testProp1".
+     * NO third party claims will be included in the idToken produced by the intermediate OP.
+     * (groupIds, testProps3, testProps4 will exist in the idToken based on the registry used by the intermed OP in the test
+     * instance)
+     *
+     * @throws Exception
+     */
+    @Test
+    public void ThirdPartyIDTokenClaims_1TestClaim_3rdPartyDoesNotPropagate() throws Exception {
+
+        String appName = "propagate1TestClaimIdTokenClaims_3rdPartyDoesNotPropagate";
+        WebConversation wc = new WebConversation();
+        TestSettings updatedTestSettings = testSettings.copyTestSettings();
+        updatedTestSettings.setTestURL(testSettings.getTestURL().replace("SimpleServlet", "simple/" + appName));
+        setParmValues(updatedTestSettings.getUserName());
+
+        List<validationData> expectations = vData.addSuccessStatusCodes(null);
+        expectations = setGroupIdsExpectations(expectations, thirdPartyPropagatedIdTokenFalse);
+        expectations = setTestParmExpectations(expectations, noThirdPartyParm, thirdPartyPropagatedIdTokenFalse);
+        expectations = setRealmNameExpectations(expectations);
+        expectations = setIssExpectations(expectations, appName);
+        expectations = setAudExpectations(expectations, appName);
+        genericRP(_testName, wc, updatedTestSettings, steps, expectations);
+    }
+
+    /**
+     * The Third party OP will populate the idToken that it creates with the groupIds claim (by default)
+     * The Third party OP does not populate the idToken that it creates with the any extra claims
+     * The intermediate OP does have thirdPartyIDTokenClaims configured with "testProp1,testProp2".
+     * NO third party claims will be included in the idToken produced by the intermediate OP.
+     * (groupIds, testProps3, testProps4 will exist in the idToken based on the registry used by the intermed OP in the test
+     * instance)
+     *
+     * @throws Exception
+     */
+    @Test
+    public void ThirdPartyIDTokenClaims_2TestClaims_3rdPartyDoesNotPropagate() throws Exception {
```

Comment:
This test is only nominally different from `ThirdPartyIDTokenClaims_1TestClaim_3rdPartyDoesNotPropagate`; I'm not sure it really adds much value.

Reply:
I want to make sure 1 vs multiple parms work without issue.